agr_java_core/src/main/java/org/alliancegenome/api/entity/GeneSummaryDocument.java

```diff
@@ -0,0 +1,24 @@
+package org.alliancegenome.api.entity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.alliancegenome.curation_api.model.entities.Gene;
+import org.alliancegenome.es.index.ESDocument;
+
+import java.util.Map;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class GeneSummaryDocument extends ESDocument {
+
+	protected String category = "gene_summary";
+	private Map<String, Object> geneAnnotations;
```

Comment:
I don't see this field being used anywhere can we remove it?

Reply:
Yes, removed.